types/raf-schd/raf-schd-tests.ts

```diff
@@ -0,0 +1,12 @@
+import rafSchd from 'raf-schd';
```

Comment:
Use the commonjs syntax `import rafSchd = require('raf-schd')`.

Consumers of the package who want to use `import rafSchd from 'raf-schd'` will need to add `"esModuleInterop": true`, but Definitely Typed tests need to be backward compatible to show that the types will work with old code bases.